tremor-pipeline/src/op/qos/backpressure.rs

```diff
@@ -32,6 +32,20 @@ use tremor_script::prelude::*;
 
 const OVERFLOW: Cow<'static, str> = Cow::const_str("overflow");
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[serde(rename_all = "lowercase")]
+pub enum Method {
+    /// messages are discarded
+    Discard,
+    /// a circuit breaker is triggerd
+    Circuitbreaker,
```

Comment:
i am not 100% sure, it is easily understood, what this option does, from the naming. What about naming it `Pause`?

Reply:
I like it!